packages/cli/src/commands/deploy/helper.ts

```diff
@@ -228,14 +231,10 @@ export const confirmOptionsUpdate = async (
     return
   }
 
-  if (options.length > 0) {
```

Comment:
I removed this console.log from here and created a new method (`lookForManifestOptions()`) for printing the options and the disclaimer because this method here is only called during updating time and I thought would be nice to add the disclaimer also during creation time.

What do you think, @eunjae-lee?